modules/storages/spec/features/storages/admin/create_storage_spec.rb

```diff
@@ -189,7 +189,7 @@
       end
     end
 
-    it "renders a Nextcloud specific multi-step form when using OAuth 2.0 SSO", :webmock, with_flag: :oidc_token_exchange do
+    it "renders a Nextcloud specific multi-step form when using OAuth 2.0 SSO", :webmock do
```

Comment:
you'll need to enable the enterprise feature for the tests

```suggestion
    it "renders a Nextcloud specific multi-step form when using OAuth 2.0 SSO", :webmock, with_ee: [:nextcloud_sso] do
```